notebook.ipynb

```diff
@@ -2,9 +2,17 @@
  "cells": [
```

Comment:
<p>In colab I was able to run for more than <code>n_sample = 20 </code> ,if sufficient ressources are available it would be great to run the same number of iterations as in the original paper for direct comparability</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eds-book-gallery/reproduce-deep-prior-4Dvar/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>I've added a note to indicate the value of <code>n_sample = 20</code> can be increased according to hardware availability.</p>

Reply:
<p>closing conversation, resolved</p>